src/Utils/Helper.php

```diff
@@ -87,7 +87,7 @@ public static function filter_input( $type, $variable_name, $filter = FILTER_DEF
 			 * Use the PHP method and bail out.
 			 */
 			switch ( $filter ) {
-				case FILTER_SANITIZE_STRING:
+				case FILTER_DEFAULT:
```

Comment:
Blank filter_var filter or FILTER_DEFAULT, is equivalent to [FILTER_UNSAFE_RAW](https://www.php.net/manual/en/filter.filters.sanitize.php). This will result in no filtering taking place by default. Filtering / sanitization should take place.

Reply:
@pdclark  `FILTER_SANITIZE_STRING` has been deprecated in PHP 8.1 and will result in E_WARNING

I think we may have to modify this function to use `sanitize_text_field` or other appropriate sanitization function.